compiler/sem/pragmas.nim

```diff
@@ -1361,29 +1249,27 @@ proc prepareSinglePragma(
           incl(sym.flags, sfBorrow)
       of wFinal:
         result = noVal(c, it)
-        if sym.typ == nil:
-          result = invalidPragma(c, it)
-        else: incl(sym.typ.flags, tfFinal)
+        assert sym.typ != nil
```

Comment:
(pre-existing issue) This should do a reciprocal check of `wInheritable` in case one pragma is applied before the other.

Reply:
Applying `wFinal` when the type is already marked as `.inheritable` is now an error.